### PR TITLE
Update contributing docs with generate/release instructions

### PR DESCRIPTION
**Pull request by lukasolson**
Should we put something in there about tagging/releasing on Github or does that happen automatically during one of these steps?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,3 +64,52 @@ Please make sure you have signed the [Contributor License Agreement](https://www
...
+```sh
+grunt version:12.0.0-rc4
+```
+
+After bumping the version, commit and push the changed files.
+
+### 2. Publish both libraries to npm and tag as needed
+
+```sh
+npm publish
```
Comment (spalger) [inline CONTRIBUTING.md]: We can do this in one line with `npm publish --tag prerelease`, which prevents the previous `latest` tag from getting changed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -64,3 +64,52 @@ Please make sure you have signed the [Contributor License Agreement](https://www
...
+npm tag elasticsearch@12.0.0-rc4 prerelease
+npm tag elasticsearch@11.0.1 latest
+cd src/bower_es_js
```
Comment (spalger) [inline CONTRIBUTING.md]: This is handled by the `browser_clients:release` grunt task

Comment (lukasolson) [inline CONTRIBUTING.md]: What exactly is handled by that?

Comment (spalger) [inline CONTRIBUTING.md]: It builds the brower versions of the client, updates the bower-elasticsearch-js repo, and publishes to npm iirc

Comment (lukasolson) [inline CONTRIBUTING.md]: So all of the stuff under step 2 here is completely unnecessary since the `browser_clients:release` is listed in step 3?

Comment (spalger) [inline CONTRIBUTING.md]: The non-browser client needs its tags pushed, but the browser client is automated

**Comment by lukasolson** [comment]
@spalger Could you take one more look? Thanksies. :)

**Pull request merged**